remote/src/main/java/com/playone/mobile/remote/PlayoneFirebase.kt

```diff
@@ -8,4 +10,22 @@ abstract class PlayoneFirebase {
     protected val FAVORITES = "favorites"
     protected val TEAMS = "teams"
     protected val NAME = "name"
+
+    abstract fun getPlayoneList(
+        userId: Int,
+        callback: (model: List<PlayoneModel>) -> Unit,
+        errorCallback: (code: Int, msg: String, detail: String) -> Unit
```

Comment:
How about use `FirebaseErrorCallback`?

Reply:
This error callback is for firebase, but I think it's too long. Might be use typealias is the best choice.